powertools-metrics/src/test/java/software/amazon/lambda/powertools/metrics/MetricsLoggerTest.java

```diff
@@ -154,13 +172,6 @@ void metricsLoggerCaptureUtilityWithDefaultNameSpace() {
         }
     }
 
-    @Test
```

Comment:
why did you remove this test ?

Reply:
Oh wow, I'm on a roll with mistakes here. Unintentional as I was moving methods around

Reply:
It's not removed, check line 138